### ISSUE TITLE
Corona reflections do not render on custom placed objects

### ISSUE THREAD

**Issue by lopezloo** [multitheftauto/mtasa-blue#2755]
### Describe the bug

Corona reflections do not render on custom placed objects.

### Steps to reproduce

```lua
crun setCoronaReflectionsEnabled(2) Object(5501, 0, 0, 10) me.position = Vector3(0, 0, 10 + 1) marker = Marker(0, 0, 10 + 1, "corona", 1.0) marker:setCoronaReflectionEnabled(true)
```

### Additional context

Similar to #450.

**Comment by lopezloo** [comment]
Most likely fixed by #3274. Feedback appreciated.

**Comment by Fernando-A-Rocha** [comment]
I just confirmed with this code that #3274 resolved this! This issue can be closed.

```lua
setElementPosition(getPedOccupiedVehicle(localPlayer) or localPlayer, 0, 0, 11)
setCoronaReflectionsEnabled(2) -- enable even during day

-- This building will reflect light
createBuilding(5501, 0, 0, 10, 0, 0, 0, 0)
local marker = createMarker(0, 0, 11, "corona", 1.0)
setCoronaReflectionEnabled(marker, true)

-- This object won't
createObject(5501, 0, 20, 10, 0, 0, 0)
local marker2 = createMarker(0, 20, 11, "corona", 1.0, 255, 255, 0)
setCoronaReflectionEnabled(marker2, true)
```

![image](https://github.com/user-attachments/assets/a6f5c1d4-4ecd-4d46-be75-8a1d5be385c2)

**Comment by Fernando-A-Rocha** [comment]
> Corona reflections do not render on custom placed objects.

Solution: use buildings instead, to have the intended behavior for world objects :P

**Comment by lopezloo** [comment]
Fixed in #3274.
